lang/java/reef-examples/src/main/java/org/apache/reef/examples/group/bgd/BGDClient.java

```diff
@@ -67,7 +67,6 @@ public BGDClient(@Parameter(InputDir.class) final String input,
    *
    * @param runtimeConfiguration the runtime to run on.
    * @param jobName              the name of the job on the runtime.
-   * @return
```

Comment:
By the way, for the `@return` annotation, our `generated codes` makes many warnings for that.

I though that's okay since we was able to get javadoc output anyway.